proposal.txt

```diff
@@ -0,0 +1,19 @@
+//Anya Lauria
+
+I want my final project to be a program that creates a character profile for the user. The character profile will be of 
+an elf, or some other fantasy creature. The user could even choose what creature they would like to create a profile of.
+The program will take user input, and I will create algorithms that take that user input and turn it into something that
+goes into the profile. So for example, when the user enters their name, the program will take the letters of their name 
+and create a new elf/hobbit/orc/witch name with it. Like that, the program will also calculate their new age, lifespan
+the realm they live in (I could include visual description), new job details, what their specialty (which is different 
+from a job, and more like a skill they work on throughout one lifetime) is. They would be informed of their race, what
+dialect they speak. I can offer an options menu for the user to choose appearance, clothes, and traits. Since all of 
+these algorithms will change with the kind of creature the user wants to be, I can use structures. I will also use loops,arrays, and I can use functions to do some of the calculations. 
+
+Additionally, if this isn't enough, I could add a short text adventure game to take them through their first day, or 
+a translating program that translates basic english phrases to the same in what new language their character speaks. Or 
+both. 
+
+This idea appeals to me as it sort of a different kind of game (i'm interested in game designing), but also because I 
+like writing, and one of my favourite parts of writing is creating new characters, and developing them as people- which
+is sort of similar to this program idea. 
```

Comment:
I like the idea of building a customized profile based on user input, and I think it would be best to add in the short text adventure game if you have the time.  Depending on how you build it, some parts of your proposal can be either very simplistic or very involved - keep in mind you only have a week (but you should make the most of it if you can!) so make sure you are able to complete at least the required portions of the project.